guides/source/contributing_to_ruby_on_rails.md

```diff
@@ -483,11 +485,20 @@ simplifies future cherry picks and keeps the git log clean.
 
 ### Update Your Branch
```

Comment:
I think it should be made explicit that you are updating your Fork, and not just a regular Branch. Therefore, I think the title should be Update Your Fork instead of Update Your Branch.